x-pack/plugins/apm/common/agent_name.ts

```diff
@@ -46,8 +49,11 @@ export const AGENT_NAMES: AgentName[] = [
   ...OPEN_TELEMETRY_AGENT_NAMES,
 ];
 
-export const isOpenTelemetryAgentName = (agentName: AgentName) =>
-  OPEN_TELEMETRY_AGENT_NAMES.includes(agentName);
+export function isOpenTelemetryAgentName(
```

Comment:
This function checks for a given agent to be an OTel agent or not, in that case, i would make the `agent` param mandatory. Anyhow we always check for the agent to be present before calling this function. You don't have to force type also if its mandatory

```
export function isOpenTelemetryAgentName(
  agentName: string
): agentName is OpenTelemetryAgentName {
  return OPEN_TELEMETRY_AGENT_NAMES.includes(agentName as AgentName);
}
```